doc/releases/release-notes-template.rst

```diff
@@ -0,0 +1,269 @@
+:orphan:
+
+.. _zephyr_X.Y:
+
+Zephyr X.Y.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version X.Y.0.
+
+.. This section is to be populated by the release manager
+
+Major features and improvements in this release include:
+
+*
+*
+
+API Changes
+***********
+
+.. Please document all API additions, removals, deprecation and changes here and
+   not in the individual subsections
+
+New APIs
+========
+
+Removed APIs
+============
+
+Deprecated APIs
+===============
+
+Changed APIs
+============
+
+.. Includes both stable and unstable/experimental API changes
+
+Security Vulnerabilities
+************************
```

Comment:
In 3.4 the security vulnerabilities part is above API changes, would it be better to keep it there due to importance?

Reply:
I thought about it but I ended up moving it. I am open to restoring its location if others agree with you.

Reply:
I think security related entries should be features very prominently in the release notes.